src/2048-expert/index.html

```diff
@@ -13,6 +13,22 @@
 <body>
     <core-navbar>2048 Expert</core-navbar>
 
+    <!-- Modal window with instructions on how to play -->
+    <core-modal>
+        <div class="modal-header">HOW TO PLAY 2048 Expert</div>
+        <p class="modal-text">
+        <ul>
+            <li>Take actions with arrow keys</li>
+            <li>Start a new game by pressing enter</li>
```

Comment:
This instruction might be redundant because the game instantly starts when the page loads

Reply:
this means there is a shortcut for start a new game with "enter" key

Reply:
This is only a "shortcut" if the New Game button is focused. If the user clicks outside of it, the button is unfocused and using the enter key to start a new game no longer works. IMO the game shouldn't restart when the user presses the "enter" key. They might be in the middle of a game and accidentally press enter (it is next to the up arrow key).